Changelog.md

```diff
@@ -1,3 +1,10 @@
+### Development
+
+Bug Fixes:
+
+* Correctly run `before(:suite)` (and friends) in the context of an example
```

Comment:
Also, technically they are run in the context of an _instance_ of an example group, not an example group.  Users are often confused by this so it's helpful if we're accurate in our changelog entries :).